test/proxy.test.js

```diff
@@ -0,0 +1,254 @@
+
+import sinon from 'sinon';
+import { ENV_VARS, LOCAL_ENV_VARS } from 'apify-shared/consts';
+import Apify from '../build/index';
+import * as requestUtils from '../build/utils_request';
+import * as utils from '../build/utils';
+
+const { apifyClient } = utils;
+
+const groups = ['GROUP1', 'GROUP2'];
+const hostname = 'proxy.apify.com';
+const port = 8000;
+const password = 'test12345';
+const country = 'CZ';
+const sessionId = 538909250932;
+const basciOpts = {
+    groups,
+    country,
+    password,
+    hostname,
+    port,
+};
+
+describe('Apify.ProxyConfiguration', () => {
+    test('should accept all options', () => {
+        const proxyConfiguration = new Apify.ProxyConfiguration(basciOpts);
+
+        expect(proxyConfiguration).toBeInstanceOf(Apify.ProxyConfiguration);
+        expect(proxyConfiguration.groups).toBe(groups);
+        expect(proxyConfiguration.country).toBe(country);
+        expect(proxyConfiguration.password).toBe(password);
+        expect(proxyConfiguration.hostname).toBe(hostname);
+        expect(proxyConfiguration.port).toBe(port);
+    });
+
+    test('should getUrl() work', () => {
```

Comment:
the point of `should` is to make the name of test readable. `should getUrl() work` is not correct. Feel free to name it `getUrl() should work` or anything else that explains the test such as `getUrl() returns proxy url`.

`should` comes from BDD (behavior driven development) where tests names would be written like

```
it('should sum two numbers', ...)
```

Reply:
Renamed. cheers.